### ISSUE TITLE
failed to init chain in NewServiceClient

### ISSUE THREAD

**Issue by sebastienturr** [seeleteam/go-seele#1080]
`?[31mERRO?[0m[0003] failed to init chain in NewServiceClient. failed to get block header, hash = 0xbce33ea5a8a2065d0792a735a2bdca4ebee
8d08cd832287eb47b1a849919e621 ===> leveldb: not found  ?[31mcaller?[0m="client.go:77" ?[31mmodule?[0m=lightclient_4
create light client manager failed. failed to get block header, hash = 0xbce33ea5a8a2065d0792a735a2bdca4ebee8d08cd832287eb47b1a849919e
621 ===> leveldb: not found
`

**Comment by shawnxin89** [comment]
> ```
> ?[31mERRO?[0m[0003] failed to init chain in NewServiceClient. failed to get block header, hash = 0xbce33ea5a8a2065d0792a735a2bdca4ebee
> 
> 8d08cd832287eb47b1a849919e621 ===> leveldb: not found  ?[31mcaller?[0m="client.go:77" ?[31mmodule?[0m=lightclient_4
> create light client manager failed. failed to get block header, hash = 0xbce33ea5a8a2065d0792a735a2bdca4ebee8d08cd832287eb47b1a849919e
> 621 ===> leveldb: not found
> ```

Are you starting multiple nodes on the same shard with the same machine?

**Comment by sebastienturr** [comment]
> > ```
> > ?[31mERRO?[0m[0003] failed to init chain in NewServiceClient. failed to get block header, hash = 0xbce33ea5a8a2065d0792a735a2bdca4ebee
> > 
> > 8d08cd832287eb47b1a849919e621 ===> leveldb: not found  ?[31mcaller?[0m="client.go:77" ?[31mmodule?[0m=lightclient_4
> > create light client manager failed. failed to get block header, hash = 0xbce33ea5a8a2065d0792a735a2bdca4ebee8d08cd832287eb47b1a849919e
> > 621 ===> leveldb: not found
> > ```
> 
> Are you starting multiple nodes on the same shard with the same machine?

No, it's when i am just on one node, trying to sync shard 2 using -m stop arg, the command stops after 10 lines, no more
